### ISSUE TITLE
is this r4 (cypress)?

### ISSUE THREAD

**Issue by ff6f8d68** [NexTre-dev/alphaver#123]
is this r4 (cypress)?

**Comment by imheavi** [comment]
No, the owner of this repository hasnt been active here for quite some time. For info about Cypress, I would go check out the wiki here 
https://alphaver.miraheze.org/wiki/AlphaVer_Wiki
